src/formats/bear-bear2bk.ts

```diff
@@ -73,4 +74,15 @@ export class Bear2bkImporter extends FormatImporter {
 			});
 		}
 	}
+
+	private removeMarkdownHeader(mdContent: string): string {
+		const lines = mdContent.split('\n');
```

Comment:
I don't think we need to split the whole file into lines just to see if the first line starts with a `#`. What if we do something like this?

```js
if (mdContent.startsWith('# ')) {
  const idx = mdContent.indexOf('\n')
  mdContent = idx > 0
    ? mdContent.substring(idx + 1)
    : ''
}
```

We should also be careful to compare against `# ` instead of `#` so that we don't pick up a tag by accident.

Reply:
Really great feedback, thank you. I've applied the changes and verified them on the checked in test file.